packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -328,8 +328,6 @@ export async function parseBuildFile() {
 export async function preTransformationUploadCode() {
     await vscode.commands.executeCommand('aws.amazonq.transformationHub.focus')
 
-    void vscode.window.showInformationMessage(CodeWhispererConstants.jobStartedNotification)
```

Comment:
We already show this in chat and are trying to move all notifications to chat anyway